examples/nas/cream/requirements.txt

```diff
@@ -0,0 +1,12 @@
+yacs
+numpy==1.17
+opencv-python==4.0.1.24
+torchvision==0.2.1
```

Comment:
Can it be upgraded?

Reply:
Yes, but I am not sure the results can be exactly reproduced. 
To ensure the results can be easily reproduced, we suggested to use the default versions, which are well tested.